core/src/main/java/org/apache/iceberg/BaseRewriteManifests.java

```diff
@@ -43,6 +45,13 @@
 
 
 public class BaseRewriteManifests extends SnapshotProducer<RewriteManifests> implements RewriteManifests {
+  private static final String INVALID_USAGE_ERR = "clusterBy/rewriteIf cannot be used with deleteManifest/addManifest";
+
+  private static final String REPLACED_CNT = "manifests-replaced";
```

Comment:
I added a static variable, so decided to fix the order of fields at the same time.

Reply:
Sounds fine. We could also fix naming to use `_COUNT` if you prefer that.